baker/DeployUtils.ts

```diff
@@ -44,6 +52,10 @@ const bakeAndDeploy = async (
                 await baker.bakeGDocPost(gdoc)
             }
         } else {
+            // co-deploy to Buildkite
+            // once we fully switch to baking on buildkite, we should await this
+            buildkite.runFullBuild(message)
```

Comment:
just to be extra sure, and so we don't get any uncaught exceptions (same above):
```suggestion
            buildkite.runFullBuild(message).catch(() => {})
```